src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
```

Comment:
- [ ] [OPTIONAL] In your src directory, we expect to see a folder `component` which will contain all your project components.